docs/PLUGIN_CATALOG.md

```diff
@@ -1,85 +1,87 @@
+| Ping | Collector | raintank |  | [snap-plugin-collector-ping](https://github.com/raintank/snap-plugin-collector-ping) | [GitHub](https://github.com/raintank/snap-plugin-collector-ping/releases/latest) |
+| Processes | Collector | Intel | Collects Linux process metrics from /proc/processes | [snap-plugin-collector-processes](https://github.com/intelsdi-x/snap-plugin-collector-processes) | [GitHub](https://github.com/intelsdi-x/snap-plugin-collector-processes/releases/latest) |
+| Procnum | Collector | raintank | No description. | [snap-plugin-collector-procnum](https://github.com/raintank/snap-plugin-collector-procnum) | [GitHub](https://github.com/raintank/snap-plugin-collector-procnum/releases/latest) |
+| Procstat | Collector | Staples-Inc | Collect process metrics using the snap telemetry framework | [snap-plugin-collector-procstat](https://github.com/Staples-Inc/snap-plugin-collector-procstat) | [GitHub](https://github.com/Staples-Inc/snap-plugin-collector-procstat/releases/latest) |
+| RabbitMQ | Collector | Intel | Collects RabbitMQ metrics from management API | [snap-plugin-collector-rabbitmq](https://github.com/intelsdi-x/snap-plugin-collector-rabbitmq) | [GitHub](https://github.com/intelsdi-x/snap-plugin-collector-rabbitmq/releases/latest) |
+| Redfish | Collector | Intel | Collects metrics from Redfish API  | In development by [@candysmurf](https://github.com/candysmurf) | No release date - [please open an issue](https://github.com/intelsdi-x/snap/issues) if you'd like to see this |
+| Scaleio | Collector | Intel | Collects ScaleIO performance metrics | [snap-plugin-collector-scaleio](https://github.com/intelsdi-x/snap-plugin-collector-scaleio) | [GitHub](https://github.com/intelsdi-x/snap-plugin-collector-scaleio/releases/latest) |
+| Smart | Collector | Intel | Collects Intel SSDs S.M.A.R.T. metrics | [snap-plugin-collector-smart](https://github.com/intelsdi-x/snap-plugin-collector-smart) | [GitHub](https://github.com/intelsdi-x/snap-plugin-collector-smart/releases/latest) |
+| Swap | Collector | Intel | Collects Linux swap metrics from /proc | [snap-plugin-collector-swap](https://github.com/intelsdi-x/snap-plugin-collector-swap) | [GitHub](https://github.com/intelsdi-x/snap-plugin-collector-swap/releases/latest) |
+| Users | Collector | Intel | Collects Linux user metrics from utmp | [snap-plugin-collector-users](https://github.com/intelsdi-x/snap-plugin-collector-users) | [GitHub](https://github.com/intelsdi-x/snap-plugin-collector-users/releases/latest) |
 
-## Committed plugins
-These plugins are in planned/active development. This list is useful if you want to reach out and contribute to the development.
+### Processors
+| Name  | Type  | Maintainer | Description | Link | Download |
+| :---- | :---- | :--------- | :---------- | :--- | :------- |
```

Comment:
Collectors have a release column while processors/publishers have a download column. Is there a reason for the difference?